lib/elixir/lib/module/checker.ex

```diff
@@ -3,53 +3,65 @@ defmodule Module.Checker do
 
   @moduledoc false
 
-  def verify(module_map, binary, cache) do
-    warnings = warnings(module_map, cache)
-    binary = chunk(binary, module_map)
+  def verify(map, chunks, cache) do
+    # TODO: Add ExCk chunk to state so that ParallelChecker does not
+    #       have to load from disk again
```

Comment:
When does the parallel checker has to load it from disk? 🤔

Reply:
Oh, it doesn't for this module since it's already preloaded in the cache. Removing this.